client/my-sites/customer-home/cards/actions/quick-links/action-box.jsx

```diff
@@ -26,6 +28,12 @@ const ActionBox = ( {
 			return iconComponent;
 		}
 
+		if ( svgIcon ) {
+			return React.cloneElement( svgIcon, {
+				className: `${ svgIcon.props.className || '' } quick-links__action-box-icon`.trim(),
```

Comment:
We should be able to use clsx here to avoid the falsy checks and trimming hassle:
```suggestion
				className: clsx(svgIcon.props.className, 'quick-links__action-box-icon'),
```